src/main/java/com/minecolonies/coremod/client/gui/WindowBuildTool.java

```diff
@@ -34,131 +34,13 @@
 import java.util.List;
 import java.util.Locale;
 
+import static com.minecolonies.api.util.constant.WindowConstants.*;
+
 /**
  * BuildTool window.
  */
 public class WindowBuildTool extends AbstractWindowSkeleton
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') class "WindowBuildTool" has 41 methods, which is greater than the 35 authorized. Split it into smaller classes. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS1448)